implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -51,6 +52,14 @@ pub struct EnrollCommand {
 
 impl EnrollCommand {
     pub fn run(self, opts: CommandGlobalOpts) {
+        if opts.global_args.output_format == OutputFormat::Json {
+            let _ = opts
+                .terminal
+                .stdout()
+                .json("The flag --output json is invalid for this command.")
+                .write_line();
+            return;
+        }
```

Comment:
To make sure that scripts can detect that this is an error, we should return an error instead of printing text to stdout.

To do that, this check should happen inside the `rpc` function, just before calling `run_impl`.

Reply:
Fixed